# Identify the current game state
vsss_msgs/GameState game_state

# Identify the team (when applicable)
vsss_msgs/TeamSide team_side

# Identify the quadrant (when applicable)
vsss_msgs/Quadrant quadrant

# Identify the period of the game (when applicable)
vsss_msgs/GamePeriod game_period
